app/pulseapi/creators/models.py

```diff
@@ -28,7 +28,7 @@ class Creator(models.Model):
     """
     name = models.CharField(max_length=140)
     user = models.ForeignKey(
-        UserProfile,
+        settings.AUTH_USER_MODEL,
```

Comment:
Probably? No idea what the idea for People pages concretely is but all this is "if it's a User, it's a User", and users exist at the Django level, not the app level. So if I logged into network-api, I have a user account that exists for every app in Django, be that network-api or pulse-api or whatever other app we end up putting beside those.